README.markdown

```diff
@@ -496,6 +496,10 @@ Generated code will be placed in the Gradle build directory.
 
 - With `--ts_proto_opt=initializeFieldsAsUndefined=false`, all optional field initializers will be omited from the generated base instances.
 
+- With `--ts_proto_opt=disableProto2Optionals=true`, all optional fields on proto2 files will not be set to be optional.
+
+- With `--ts_proto_opt=disableProto2DefaultValues=true`, all fields in proto2 files that specify a default value will not actually use that default value.
```

Comment:
@lukealvoeiro wdyt about adding a rationale/explanation to these like "This flag is primarily for preserving ts-proto's original handling of proto2 files, to avoid breaking changes".

Just thinking new users might read both of these and wonder "why/when should I set this?".

Reply:
Great idea. Just added it!